dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -153,68 +149,62 @@ private Response postWithEndpointState(String endpointId, String url, MediaType
 		return response;
 	}
 
+    private String resolveRemoteIp(@Context HttpServletRequest request) {
+        String remoteIP;
+        remoteIP = request.getRemoteHost();
+        if (!UtilMethods.isSet(remoteIP))
+            remoteIP = request.getRemoteAddr();
+        return remoteIP;
+    }
 
     /**
      * <p>Returns a zip with data from structures and folders for integrity check
-     *
-     * Usage: /getdata
-     *
      */
-
     @POST
     @Path("/generateintegritydata/{params:.*}")
     @Consumes(MediaType.MULTIPART_FORM_DATA)
     @Produces("text/plain")
-    public Response generateIntegrityData(@Context HttpServletRequest request, @FormDataParam("AUTH_TOKEN") String auth_token_digest)  {
+    public Response generateIntegrityData(
+            @Context final HttpServletRequest request,
+            @FormDataParam("AUTH_TOKEN") final String auth_token_digest)  {
```

Comment:
Codacy found an issue: [The String literal "AUTH_TOKEN" appears 8 times in this file; the first occurrence is on line 169](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)